en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,296 @@
+        Successfully logged into Microgateway Adapter in environment: <environment-name>
+        ```
+
+        ```bash tab="Example Response"
+        Successfully logged into Microgateway Adapter in environment: dev
+        ```
+    
+    !!! warning
+        Using `--password` in CTL is not secure. Instead, you can use `--password-stdin` to include password file or provide the password when prompted. 
+
+## Deploy an API
+
+In order to deploy, let us first create an API project. Follow the steps in [Initialize an API Project]({{base_path}}/install-and-setup/setup/api-controller/managing-apis-api-products/importing-apis-via-dev-first-approach/#initialize-an-api-project) to create an API project using **apictl**.
+
+An API project can be deployed to a microgateway using the following commands.
+
+-   **Command**
+    ```go
+    apictl mg deploy api -e <environment-name> -f <api-project-file-path> -k
+    ```
+    ```go
+    apictl mg deploy api ---environment <environment-name> --file <api-project-file-path> -k
+    ```
+
+    !!! tip
+        In an API project the API name and the API version given in the `api.yaml` will be taken as the name and version of the API once deployed.
+
+    !!! tip
+        When trying to deploy, if an API with the same name, version, virtual host combination already exists in the microgateway adapter, an error response will be given saying that the API already exists.
+
+        In order to override the existing API or deploy an API without considering whether it already exists or not, include the flag `--override` or its shorthand `-o` to the same command above. For an example,
+        ```bash
+        apictl mg deploy api -e dev -f petstore -o -k
+        ```  
+
+    !!! info
+        **Flags:**
+        
+        - Mandatory :    
+            `-e`, `--environment` : Microgateway adapter environment to add the API   
+            `-f`, `--file`        : Filepath of the apictl project to be deployed
+
+        - Optional :     
+            `-o`, `--override` : Whether to deploy an API irrespective of its existance. Overrides when exists.    
+            `--skip-cleanup` : Whether to keep all temporary files created during the deploy process
+
+    !!! example
+        ```bash
+        apictl mg deploy api -e dev -f petstore -k
+        ```
+        ```bash
+        apictl mg deploy api --environment dev --file petstore -k
+        ```
+        ```bash
+        apictl mg deploy api -e dev -f petstore --override -k
+        ```
+        ```bash
+        apictl mg deploy api -e dev -f petstore -o --skip-cleanup -k
+        ```
+
+-   **Response**
+
+    ``` bash tab="Deployed"
+    Successfully deployed API to microgateway.
+    ```
+
+    ``` bash tab="API Already Exists"
+    Unable to deploy API. API already exists. Status: Conflict 409
+    ```
+
+    ```bash tab="Deployed or Updated"
+    Successfully deployed/updated the API in microgateway.
+    ```
+
+## List Deployed APIs
+
```

Comment:
Shall we add an introduction sentence like "**This command uses to list the deployed APIs on a given microgateway  environment"**  ?

Reply:
Here as well,
"This command can be uses to..." would be more suitable WDYT? @Chamindu36

Reply:
+1

Reply:
Adding the intro line "This command can be used to list the deployed APIs on a given microgateway adapter environment."